frontend/start-dev.js

```diff
@@ -35,7 +35,7 @@ if (startDevProxy) {
 }
 
 process.stdout.write("Starting @rotki/common watch \n");
-const commonProcesses = spawn("npm run watch -w @rotki/common", {
+const commonProcesses = spawn("sleep 30 && npm run watch -w @rotki/common", {
```

Comment:
Why is this included in this PR?

Reply:
because it wouldn't make sense to make a new PR only for that now would it? :P
There was a problem where without this the common watch will happen in parallel with the app-start and cause some build errors